shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -58,6 +58,8 @@ class FlutterPlatformViewsController {
 
   void PrerollCompositeEmbeddedView(int view_id);
 
+  NSObject<FlutterPlatformView>* GetPlatformViewByID(int view_id);
```

Comment:
Add a doc comment.

Reply:
done

Reply:
nit: use backticks around type names to help the docgen tool we use (Jazzy) generate links etc.